src/components/PokemonList/NotFound.tsx

```diff
@@ -0,0 +1,18 @@
+import { View, Text, StyleSheet } from "react-native";
+
+const NotFound = () => {
```

Comment:
Why do you prefer 
```ts
const NotFound = () => {
...
export default NotFound;

```
over 
```ts
export default function NotFound() {
```

Reply:
the only reason is that I started writing in this way:

```
const NotFound = () => {
...
export default NotFound;
```

in the next project, if I have to make a decision I will choose `export default function NotFound() {`

Reply:
I was just curious what is your reason behind it. You can use whatever you want (or whatever is preferred in project you are working on)